specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations-authoring.json

```diff
@@ -0,0 +1,2703 @@
+          "format": "date-time",
+          "description": "Gets or sets the last updated time.",
+          "type": "string"
+        },
+        "expirationDateTime": {
+          "format": "date-time",
+          "description": "Gets or sets the expiration time.",
+          "type": "string"
+        },
+        "status": {
+          "description": "Gets or sets the job status.",
+          "enum": [
+            "unknown",
+            "notStarted",
+            "running",
+            "succeeded",
+            "failed",
+            "cancelled",
+            "cancelling",
+            "partiallyCompleted"
+          ],
+          "type": "string"
+        },
+        "errors": {
+          "description": "Gets or sets the errors.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/JobErrorObject"
+          }
+        }
+      }
+    },
+    "ProjectFileExportObject": {
+      "required": [
+        "assetsToExport"
+      ],
+      "type": "object",
+      "properties": {
+        "assetsToExport": {
+          "description": "Gets or sets the assets to export.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "ProjectFileExportJobStatusObject": {
+      "required": [
+        "jobId",
+        "createdDateTime",
+        "lastUpdatedDateTime",
+        "expirationDateTime",
+        "status",
+        "errors"
+      ],
+      "type": "object",
+      "properties": {
+        "resultUrl": {
+          "type": "string"
+        },
+        "jobId": {
+          "description": "Gets or sets the jobId.",
+          "type": "string"
+        },
+        "createdDateTime": {
+          "format": "date-time",
+          "description": "Gets or sets the created time.",
+          "type": "string"
+        },
+        "lastUpdatedDateTime": {
+          "format": "date-time",
+          "description": "Gets or sets the last updated time.",
+          "type": "string"
+        },
+        "expirationDateTime": {
+          "format": "date-time",
+          "description": "Gets or sets the expiration time.",
+          "type": "string"
+        },
+        "status": {
+          "description": "Gets or sets the job status.",
+          "enum": [
+            "unknown",
+            "notStarted",
+            "running",
+            "succeeded",
+            "failed",
+            "cancelled",
+            "cancelling",
+            "partiallyCompleted"
+          ],
+          "type": "string"
+        },
+        "errors": {
+          "description": "Gets or sets the errors.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/JobErrorObject"
+          }
+        }
+      }
+    },
+    "CluProjectCreateObject": {
+      "required": [
+        "type",
+        "settings",
+        "name",
+        "language"
+      ],
+      "type": "object",
+      "properties": {
+        "type": {
```

Comment:
Be sure to check all instances of this, but just remembering now...

1. We prefer "kind" over "type" in the Azure SDK, since "type" can be quite overloaded, especially in strictly-typed languages.
2. You can avoid renaming this service-side by using an `"x-ms-client-name": "kind"`.
3. This should be a more describe name anyway, like "projectKind" so it's more obvious to the user. The inference swagger already does this: https://github.com/Azure/azure-rest-api-specs/blob/df7bcbefb2993484476a496eea600c3dc9777c65/specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json#L299-L309